Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty.py

```diff
@@ -263,17 +357,16 @@ def update_threat_intel_set(client: boto3.client, args: dict):
                         f"Response was: {response}")
 
 
-def severity_mapping(severity: int):
-    if severity <= 3.9:
-        demistoSevirity = 1
-    elif severity >= 4 and severity <= 6.9:
-        demistoSevirity = 2
-    elif severity >= 7 and severity <= 8.9:
-        demistoSevirity = 3
-    else:
-        demistoSevirity = 0
-
-    return demistoSevirity
+def severity_mapping(severity: Optional[int]):
```

Comment:
If we are getting `severity = None` we want to return `demisto_severity = None`?